datasets/standvastgoed/dataset.json

```diff
@@ -49,85 +49,85 @@
             "title": "id",
             "description": "Unieke identificatie voor dit object op basis van identificatie, volgnummer en geldigheid"
           },
-          "standBeginGeldigheid": {
+          "standBegingeldigheid": {
             "type": "string",
             "format": "date-time",
-            "title": "standBeginGeldigheid",
-            "shortname": "standBeginGeldigheid",
+            "title": "standBegingeldigheid",
+            "shortname": "standBegingeldigheid",
```

Comment:
A shortname that is equal to the actual id of the field is a bit weird. Better to remove it.